site/src/components/ParameterInput/ParameterInput.tsx

```diff
@@ -1,4 +1,5 @@
 import FormControlLabel from "@material-ui/core/FormControlLabel"
+import MenuItem from "@material-ui/core/MenuItem"
 import Radio from "@material-ui/core/Radio"
 import RadioGroup from "@material-ui/core/RadioGroup"
```

Comment:
RadioGroup is not being used anymore. Suggested to remove import.

Reply:
It is for the boolean types